src/main/java/ru/itis/security/filters/JwtRequestFilter.java

```
@@ -40,12 +42,12 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
             try {
                 username = jwtTokenUtil.getUsernameFromToken(jwtToken);
             } catch (IllegalArgumentException e) {
-                logger.error("Unable to get JWT Token");
+                log.error("Unable to get JWT Token");
```

Don't ignore exceptions